### PR TITLE
build: update to latest versions of txauthor and txsizes

### PR DESCRIPTION
**Pull request by Roasbeef**
This fixes an error in taproot based fee estimation.

### CODE REVIEW

File: wallet/txauthor/author.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func NewUnsignedTransaction(outputs []*wire.TxOut, feeRatePerKb btcutil.Amount,
 
 	targetAmount := SumOutputValues(outputs)
 	estimatedSize := txsizes.EstimateVirtualSize(
-		0, 1, 0, outputs, changeSource.ScriptSize,
+		0, 0, 1, 0, outputs, changeSource.ScriptSize,
```
Comment (guggero) [inline wallet/txauthor/author.go:98]: hmm, unfortunately this change won't take effect until we push yet another version of the `txauthor` package.

Comment (Roasbeef) [reply wallet/txauthor/author.go:98]: Ah true, since we updated `txauthor` in the past PR that added this to `txsizes`...

**Comment by Roasbeef** [comment]
Ok I made #813 that fixed the issue in the package. 

I thought we fixed this in the past, but we need to start building/running tests for each of the modules individually. Otherwise, if we update a sub-module, then we just run the tests from the PoV main module, which depends on an earlier version of the module, so this change isn't detected.

**Comment by Roasbeef** [comment]
Ok, so the new path is:
  * Merge #814 -- tag a new version 
  * Fix the `txauthor` package, update to above version 
  * Merge that -- tag new version
  * Update this PR after the steps above are done

**Review by guggero — APPROVED** [review]

**Pull request merged**